web/src/components/Content.test.tsx

```diff
@@ -44,13 +44,6 @@ describe("<Content />", () => {
     });
   });
 
-  it("renders greenBox content", () => {
-    const mdString = ":::greenBox\n" + "body text\n" + ":::";
-
-    render(<Content>{mdString}</Content>);
-    expect(screen.getByText("body text")).toBeInTheDocument();
-  });
-
```

Comment:
[sand] Should this get replaced with a test that checks the `<Callout />` gets substituted?

Reply:
I don't think so, the green box syntax is getting deleted all together